src/Builder/Visitor/AddPluginToPluginListVisitor.php

```diff
@@ -372,4 +408,12 @@ protected function successfullyProcessed(): int
 
         return NodeTraverser::DONT_TRAVERSE_CHILDREN;
     }
+
+    /**
+     * @return \SprykerSdk\Integrator\Builder\Visitor\PluginPositionResolver\PluginPositionResolverInterface
+     */
+    protected function getPluginPositionResolver(): PluginPositionResolverInterface
+    {
+        return new PluginPositionResolver();
+    }
```

Comment:
And here also. I think DI in constructor for PluginPositionResolver will be better

Reply:
++